### PR TITLE
Revert Explorer to Gateway Explorer not Mechanic-lite Explorer

### PR DESCRIPTION
**Pull request by CkeyLurapa**
**## What Does This PR Do**
Returns Explorer to Gateway Explorer.

**## Why It's Good For The Game**
This PR uses the logic of other Karma jobs. That one cannot get transferred into a job they do not have unlocked or is barred (such as Magistrate). And any lite-mimics/imitation of the Job is not advised or repealed in-game.

The Explorer used to be custom-made and often only had Gateway/EVA. Not External. While Mechanic's job is not to solely Explore Space. It is what they do 9/10. With a Pod, of course. 

This argument that the Explorer and Mechanic are different because one has a pod, and the other does not. Is odd. Because lite-Blueshields do not have Combat Gloves or proper equipment. But still get bwoinked/reeled in. Magistrate-lite may not have CC authority but still get reeled in. 

Upon returning to the game. The fact Explorers reek so much of Mechanic, yet seems absent of Karma Job protections is odd to me.

Furthermore, it'll help Security reel in Explorers on Blue/Red. Who should be out of a job with Gateway shut down.
**## Images of changes**
image

**## Changelog**
:cl:
tweak: removed external-airlock from Explorer job.
/:cl:

### CODE REVIEW

**Comment by datlo** [comment]
I mean, sure, but this has little relevance when there's an airlock that's public access in arrivals.

**Comment by hal9000PR** [comment]
Mechanics job isn't to explore space though, the mechanics job is to make space pods for explorers.

**Comment by CkeyLurapa** [comment]
The issue is that Karma Job, Mechanic. Can be attained / replicated with or without having it unlocked. Is the issue I have here.

Moreso if the Mechanic and bestow pods upon Explorers. Therefore making Explorers a Mechanic-lite, without the unlock. And the extra accesses a Mechanic has.

**Comment by Kyep** [comment]
Objection.

When I added Explorer, it was meant to explore both Gateway and space.
If it was gateway only, I would have named it "Gateway Explorer". I would not have given it external airlock access either.

**Comment by BryanPara** [comment]
I don't see an issue with it being Mechanic-lite? It isn't Mechanic, they don't get access to the garage and are unable to build pods. If anything this gives the Mechanic individuals who would seek them out for the fact that they can make pods to more easily facilitate exploration.

Mechanics were never intended to be space explorers. Hence why they were named Mechanic and not Space Explorer. The intent was to make pods for whoever wanted one to go derping about in space with it, which Explorer now gives them a group of customers so to speak.

**Review by Kyep — COMMENTED** [review]
Objection.

When I added Explorer, it was meant to explore both Gateway and space.
If it was gateway only, I would have named it "Gateway Explorer". I would not have given it external airlock access either.

**Comment by Spacemanspark** [comment]
Mechanics make pods, explorers use pods to go explore.
:-1:

**Comment by AffectedArc07** [comment]
image
Calling this a revert is dishonest. They were never only gateway explorers. This is a tweak. The mechanic's job also isn't to go out and loot hunt. They are a spacepod creator primarily, with space exploration as their side job. Confining space to one role isn't great, and should not be done. 

TLDR: 👎

**Comment by Mitchs98** [comment]
I'm not quite sure what the issue here is whatsoever, really? Just because they have access to external airlocks doesn't make them mechanic-lite or anything of that caliber by any means remotely. The access doesn't make the job, really, at-least in this case. This is a far cry from someone in command making personal body guards or what not to mimic Blueshields. They have no access to anything that could be considered mechanic-y.

Either way as stated in a previous comment, the whole argument is moot when there's a public access airlock.

**Comment by procdrone** [comment]
If anything, the issue here is mechanic not doing his job and simply using it for power space explorer (most often, mechanic is picked by people aiming to raid syndie depot).
